src/execution/execute.js

```diff
@@ -378,9 +378,9 @@ export function buildExecutionContext(
  */
 function executeOperation(
   exeContext: ExecutionContext,
-  operation: OperationDefinitionNode,
```

Comment:
The existing function signature was easier to understand - `executeOperation()` should accept an `operation`.